### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cssuh**

### CODE REVIEW

**Comment by abist** [comment]
Please change the date in the changelog as well

**Review by abist — APPROVED** [review]
Merge after updating the date in changelog too

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ See [the SQL developer tutorial] to develop an app with C#, Java, Node.js, PHP,
 <img src="https://github.com/Microsoft/vscode-mssql/raw/main/images/mssql-demo.gif" alt="demo" style="width:480px;"/>
 
 ## Version 1.10.0
-* Release date: November 4, 2020
+* Release date: December 8, 2020
```
Comment (kburtram) [inline README.md:25]: should this be December 10, since that's the planned release date?

**Review by kburtram — APPROVED** [review]
Looks good, but I'd use the actual release date if possible.

**Pull request merged**